### ISSUE TITLE
TypeError: LoadLibrary() argument 1 must be string, not unicode

### ISSUE THREAD

**Issue by davemasterdave** [pywinauto/pywinauto#283]
Hi, I am running into the issue below when trying to import version 0.5.2+ of pywinauto. Version 0.5.1 works without any issue so I am using that version for now. On a sidenote, this project is a lifesaver!!
An import statement such as
```python
from pywinauto import application
```
results in:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python27\lib\site-packages\pywinauto\__init__.py", line 40, in <module>
    from . import findwindows
  File "C:\Python27\lib\site-packages\pywinauto\findwindows.py", line 39, in <module>
    from . import win32functions
  File "C:\Python27\lib\site-packages\pywinauto\win32functions.py", line 216, in <module>
    shcore = ctypes.windll.LoadLibrary(u"Shcore.dll")
  File "C:\Python27\lib\ctypes\__init__.py", line 440, in LoadLibrary
    return self._dlltype(name)
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
TypeError: LoadLibrary() argument 1 must be string, not unicode
```
My system is Windows 10 64 bit and my interpreter is
```
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
```

**Comment by airelil** [comment]
Please check the PR #281, we just merged yesterday. Seems as your issue

**Comment by davemasterdave** [comment]
Ok sounds good. When do you think the next release will come out?

**Comment by airelil** [comment]
We still have a bunch of issues to close before the release 0.6.1 and we are slowed down for the holidays. You can try it out though just by replacing ```win32functions.py``` temporarily. If you go on it, please, let us know if the fix helps.

**Comment by vasily-v-ryabov** [comment]
I hope we will finish with 0.6.1 around end of January. Though some fixes might be postponed till 0.6.2.

**Comment by vasily-v-ryabov** [comment]
This was fixed and auto-tested in master branch (AppVeyor also switched to Python 2.7.13 and I saw the issue on CI, it was gone after the fix). I'm closing the issue. pywinauto 0.6.1 seems coming on time (till end of January).
